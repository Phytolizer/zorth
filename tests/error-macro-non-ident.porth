macro 1 end
