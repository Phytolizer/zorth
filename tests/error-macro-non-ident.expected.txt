:i returncode 1
:b stdin 0

:b stdout 0

:b stderr 100
tests/error-macro-non-ident.porth:1:7: ERROR: expected macro name to be a word but found an integer

